### PR TITLE
feat(tests): added wasm tests

### PR DESCRIPTION
**Pull request by cnlangzi**

### CODE REVIEW

File: .github/workflows/manual_system_tests.yml [hunk 1]
```diff
@@ -254,32 +254,32 @@ jobs:
...
+    #     zdns_image: ${{ env.ZDNS_TAG }}
+    #     zbox_cli_branch: ${{ inputs.zbox_cli_branch }}
+    #     zwallet_cli_branch: ${{ inputs.zwallet_cli_branch }}
+    #     zs3_server: ${{ inputs.zs3server_branch }}
+    #     custom_go_sdk_version: ${{ env.GOSDK_SHA }}
+    #     TENDERLY_FORK_ID: ${{ secrets.TENDERLY_FORK_ID }}
+    #     SUBGRAPH_API_URL: ${{ secrets.SUBGRAPH_API_URL }}
+    #     zs3_client: ${{ inputs.zs3server_branch }}
+    #     zs3_logsearchapi: ${{ inputs.zs3server_branch }}
+    #     zs3_minio: ${{ inputs.zs3server_branch }}
```
Comment (Kishan-Dhakan) [inline .github/workflows/manual_system_tests.yml:278]: Needs to be reverted

File: run-system-tests/action.yml [hunk 2]
```diff
@@ -141,14 +135,22 @@ runs:
         echo "INSTALL REQUIRED DEPENDENCIES"
         echo "======================================================"
         echo
-        echo "installing youtube-dl..."
+        curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/yarn.gpg
```
Comment (Kishan-Dhakan) [inline run-system-tests/action.yml:138]: Please echo whatever this is trying to do

**Review by Kishan-Dhakan — DISMISSED** [review]
Needs cleanup but lgtm otherwise

File: run-system-tests/action.yml (hunk 2 above)
Comment (cnlangzi) [reply run-system-tests/action.yml:139]: fixed

File: .github/workflows/manual_system_tests.yml (hunk 1 above)
Comment (cnlangzi) [reply .github/workflows/manual_system_tests.yml]: fixed

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**